solutions/usaco-811.mdx

```diff
@@ -229,4 +229,41 @@ public class snowboot {
 ```
 
 </JavaSection>
+
+<PySection>
+
+```python
+with open('snowboots.in') as r:
+	n, b = map(int, r.readline().split())
+	depth = list(map(int, r.readline().split()))
+	max_depth = [[*map(int, r.readline().split())] for _ in range(b)]
+
+stor = [[0] * b for _ in range(n)]
+stor[0][0] = 1
+for i in range(n):
+	cur = -1
```

Comment:
It's more clear if we use `None` instead of `-1` here, since `None` explicitly conveys that there is nothing found yet.

It may be true that `cur` can never be negative, but this is an extra step of reasoning someone has to make in order to understand the code.

Reply:
could I just explain this in a comment? in line 256, it's simpler and easier if cur is set it -1, rather than None.

Reply:
I missed how `cur` was being used later when reading the code. I think the original code is fine now that I realize this.

You can add a comment if you want. I think it's fine either way.